CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
-Contributions are welcomed via pull requests on Bitbucket. First contact the HOOMD-blue developers prior to beginning
+Contributions are welcomed via pull requests on Github. First contact the HOOMD-blue developers prior to beginning
```

Comment:
You could replace this with a link if desired, like "via [pull requests on GitHub](https://github.com/glotzerlab/hoomd-blue/pulls)." (also capital H in GitHub)

Similarly below.